src/components/DonationForm/Forms/UpgradeToMonthlyForm.vue

```diff
@@ -0,0 +1,177 @@
+					>
+						<label class="t-annual-upgrade-no">
+							<input
+								tabIndex="-1"
+								type="radio"
+								v-model="interval"
+								name="alternative"
+								:value="Intervals.ONCE.value"
+								class="wmde-banner-select-group-input"
+								@change="onChange"
+							/>
+							<span class="wmde-banner-select-group-label">
+                                {{ $translate( 'upgrade-to-monthly-no', { amount: secondPageAmount } ) }}
+                            </span>
+						</label>
+					</div>
+					<div
+						class="wmde-banner-select-group-option wmde-banner-select-group-option-yes"
+						:class="{ 'active': interval === Intervals.MONTHLY.value }"
+					>
+						<label class="t-annual-upgrade-yes">
+							<input
+								tabIndex="-1"
+								type="radio"
+								v-model="interval"
+								name="alternative"
+								:value="Intervals.MONTHLY.value"
+								class="wmde-banner-select-group-input"
+								@change="onChange"
+							/>
+							<span class="wmde-banner-select-group-label">
+                                {{ $translate( 'upgrade-to-monthly-yes', { amount: secondPageAmountMonthlyFraction } ) }}
+                            </span>
+						</label>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<a
+			tabIndex="-1"
+			href="#"
+			class="wmde-banner-form-upgrade-custom t-annual-upgrade-yes-custom"
+			:class="{ 'wmde-banner-form-upgrade-custom--with-error': intervalValidity === Validity.Invalid }"
+			@click.prevent="onGoToChangeOfAmount"
+		>
+			{{ $translate( 'upgrade-to-monthly-link' ) }}
+		</a>
+
+		<span v-if="intervalValidity === Validity.Invalid" class="wmde-banner-select-group-error-message">
+            <span class="wmde-banner-error-icon">
+                {{ $translate( 'upgrade-to-monthly-error' ) }}
+            </span>
+        </span>
+
+		<div class="wmde-banner-form-button-container upgrade-to-yearly-button">
+			<button tabIndex="-1" class="wmde-banner-form-button t-submit-upgrade-to-yearly" type="submit">
+				{{ $translate( 'upgrade-to-monthly-button' ) }}
+			</button>
+		</div>
+
+	</form>
+</template>
+
+<script lang="ts">
+// All form components must have names
+export default {
+	name: 'UpgradeToMonthlyForm'
+};
+</script>
+<script setup lang="ts">
+
+import { inject, ref, watch } from 'vue';
+import { useFormModel } from '@src/components/composables/useFormModel';
+import { Validity } from '@src/utils/FormModel/Validity';
+import ChevronLeftIcon from '@src/components/Icons/ChevronLeftIcon.vue';
+import { Intervals } from '@src/utils/FormItemsBuilder/fields/Intervals';
+import { Currency } from '@src/utils/DynamicContent/formatters/Currency';
+import { Tracker } from '@src/tracking/Tracker';
+import { UpgradeToMonthlyEvent } from '@src/tracking/events/UpgradeToMonthlyEvent';
+import { useFormStepShownEvent } from '@src/components/DonationForm/Forms/useFormStepShownEvent';
+
+interface AmountSuggestion {
+	lowerIntervalLimit: number,
```

Comment:
The naming here is a bit confusing because we have a separate interval field in our form. I would call them `lowerRangeLimit` and `upperRangeLimit`